ha/core/config/config_manager.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
year 2021. Please change in every new source file.

Reply:
@ajaysrivas 
is this change only need for new file added or all files from main?

if it is for all main file can we have common pr for this?

Reply:
only new files.

Reply:
Done